hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/IncrementalContainerReportHandler.java

```diff
@@ -88,8 +88,10 @@ public void onMessage(final IncrementalContainerReportFromDatanode report,
             // Ensure we reuse the same ContainerID instance in containerInfo
             id = container.containerID();
           } finally {
-            if (!replicaProto.getState().equals(
+            if (replicaProto.getState().equals(
                 ContainerReplicaProto.State.DELETED)) {
+              nodeManager.removeContainer(dd, id);
+            } else {
```

Comment:
Is this change brought in accidentally by the merge conflict resolution?

https://github.com/apache/ozone/pull/4125/commits/b2bac5584d575dbf02eb1d52628a19a6034f2a81

Reply:
@smengcl Yes, and the changes in `TestIncrementalContainerReportHandler` as well.

Reply:
Could be a GitHub PR diff display bug. See https://github.com/apache/ozone/pull/4134/files#r1094909600

Reply:
@smengcl What if I try another rebase?

Reply:
Another merge, fixed it.